RichEditorView/Classes/RichEditorView.swift

```diff
@@ -6,145 +6,189 @@
 }
 
 /// RichEditorView is a UIView that displays richly styled text, and allows it to be edited in a WYSIWYG fashion.
-@objcMembers open class RichEditorView: UIView, UIScrollViewDelegate, UIWebViewDelegate, UIGestureRecognizerDelegate {
 
+@objcMembers open class RichEditorView: UIView, UIScrollViewDelegate, UIGestureRecognizerDelegate, WKURLSchemeHandler {
+    
+    
+    
     // MARK: Public Properties
-
+    
     /// The delegate that will receive callbacks when certain actions are completed.
     open weak var delegate: RichEditorDelegate?
-
+    var task: WKURLSchemeTask?
+    
     /// Input accessory view to display over they keyboard.
     /// Defaults to nil
     open override var inputAccessoryView: UIView? {
-        get { return webView.cjw_inputAccessoryView }
-        set { webView.cjw_inputAccessoryView = newValue }
+        get { return webView.accessoryView }
+        set {webView.accessoryView = newValue}
+        
     }
-
-    /// The internal UIWebView that is used to display the text.
-    open private(set) var webView: UIWebView
-
+    
+    /// The internal WKWebView that is used to display the text.
+    open private(set) var webView: RichEditorWebView
+    
     /// Whether or not scroll is enabled on the view.
     open var isScrollEnabled: Bool = true {
         didSet {
             webView.scrollView.isScrollEnabled = isScrollEnabled
         }
     }
-
+    
     /// Whether or not to allow user input in the view.
     open var isEditingEnabled: Bool {
         get { return isContentEditable }
         set { isContentEditable = newValue }
     }
-
+    
     /// The content HTML of the text being displayed.
     /// Is continually updated as the text is being edited.
     open private(set) var contentHTML: String = "" {
         didSet {
             delegate?.richEditor?(self, contentDidChange: contentHTML)
         }
     }
-
+    
     /// The internal height of the text being displayed.
     /// Is continually being updated as the text is edited.
     open private(set) var editorHeight: Int = 0 {
         didSet {
             delegate?.richEditor?(self, heightDidChange: editorHeight)
         }
     }
-
+    
     /// The value we hold in order to be able to set the line height before the JS completely loads.
     private var innerLineHeight: Int = 28
-
+    
     /// The line height of the editor. Defaults to 28.
-    open private(set) var lineHeight: Int {
-        get {
-            if isEditorLoaded, let lineHeight = Int(runJS("RE.getLineHeight();")) {
-                return lineHeight
-            } else {
-                return innerLineHeight
+    open private(set) var lineHeight: Int = 28 {
+        didSet {
+            innerLineHeight = lineHeight
+            webView.evaluateJavaScript("RE.setLineHeight('\(innerLineHeight)px');") {  (response: Any?, error: Error?) in
+                if let e = error {
+                    print(e)
```

Comment:
Where does that print?

Reply:
I think i forgot t remove that  when debugging... will remove it.